src/Hedgehog/Gen.fs

```diff
@@ -368,13 +355,13 @@ module Gen =
 
     // Generates a random character in the specified range.
     let charRange (lo : char) (hi : char) : Gen<char> =
-        range (int lo) (int hi) |> map char
+        integral <| Range.constant (int lo) (int hi) |> map char
 
     /// Generates a random character.
     let char : Gen<char> =
         let lo = System.Char.MinValue
         let hi = System.Char.MaxValue
-        sizedRange (int lo) (int hi) |> map char
```

Comment:
This has changed the semantics, as it used to generate different characters depending on the size